core/src/main/java/com/linecorp/armeria/client/ClientOptions.java

```diff
@@ -129,7 +129,6 @@
             ClientOption.define("ENDPOINT_REMAPPER", Function.identity());
 
     private static final List<AsciiString> PROHIBITED_HEADER_NAMES = ImmutableList.of(
-            HttpHeaderNames.CONNECTION,
```

Comment:
We still need to prohibit this header if its value is not `close`.

Reply:
Good point!